package.json

```diff
@@ -6,5 +6,8 @@
   "devDependencies": {
     "markdown-toc": "1.2.0",
     "renovate": "37.374.1"
+  },
+  "overrides": {
+    "autolinker": ">= 4.0.0 ^4.0.0"
```

Comment:
I believe this can be just `^4.0.0` as the set of versions selected by `^4.0.0` is a subset of those selected by `>=4.0.0` (you can play around with this on https://semver.npmjs.com/).

Reply:
You're right. changed.